### PR TITLE
Migrating testonly views to be Injectable

### PR DESCRIPTION
**Pull request by chandraAndem**

### CODE REVIEW

File: app/testOnly/views/helpers/FieldErrorHelper.scala.html [hunk 1]
```diff
@@ -18,6 +18,8 @@
 @import play.api.data.Field
 @import testOnly.forms.validation.ErrorMessageHelper._
 
+@this()
```
Comment (JSimmons072019) [inline app/testOnly/views/helpers/FieldErrorHelper.scala.html:21]: Why is this here

Comment (chandraAndem) [reply app/testOnly/views/helpers/FieldErrorHelper.scala.html:21]: @this() is how you make a template injectable. With or without arguments.

**Review by JSimmons072019 — APPROVED** [review]
